change/@microsoft-fast-element-7edb2387-fe6a-4f0b-a596-833b26d8298d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fill color design token update",
+  "packageName": "@microsoft/fast-element",
+  "email": "skawian@gmail.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
This change to fast-element should likely be a "none" change. I'm not sure we need to update if this MD file is the only change.